ofproto/ofproto-dpif-xlate.c

```diff
@@ -3001,17 +3053,45 @@ is_ip_local_multicast(const struct flow *flow, struct flow_wildcards *wc)
 }
 
 #if defined(P4OVS)
+static enum p4_vlan_mode
+get_p4_vlan_mode(enum port_vlan_mode vlan_mode) {
+    if (vlan_mode == PORT_VLAN_ACCESS)
```

Comment:
Why not use a `switch` statement?

Reply:
Yes, we can use switch case as well for easily readability. Will make a note on this one.